src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -689,8 +689,7 @@ uct_dc_mlx5_ep_flush_remote(uct_dc_mlx5_ep_t *ep, uct_completion_t *comp)
 
 static UCS_F_ALWAYS_INLINE int uct_dc_ep_is_flush_remote(uct_dc_mlx5_ep_t *ep)
 {
-    return ucs_test_all_flags(ep->flags, UCT_DC_MLX5_EP_FLAG_FLUSH_RKEY |
-                                         UCT_DC_MLX5_EP_FLAG_FLUSH_REMOTE);
+    return ep->flags &UCT_DC_MLX5_EP_FLAG_FLUSH_REMOTE;
```

Comment:
1. code format, pls check clang-format
2. do we really need this wrapper?

Reply:
> * code format, pls check clang-format
> * do we really need this wrapper?

removed wrapper, it was added to "unify" code with RC